controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -82,7 +106,9 @@ def run_match(supervisor: Supervisor) -> None:
 
 
 def main() -> None:
-    quit_if_development_mode()
+    # quit_if_development_mode()
```

Comment:
le comment

Reply:
nothing to see here. 794df703bf336bedb7ec4d8611c4760fbf985175